uint8 channel
uint16 pulse_duration_us
---
bool result
